docs/data/material/components/tables/tables.md

```diff
@@ -115,7 +115,7 @@ A simple example with spanning rows & columns.
 
 ## Virtualized table
 
-In the following example, we demonstrate how to use [react-virtualized](https://github.com/bvaughn/react-virtualized) with the `Table` component.
+In the following example, we demonstrate how to use [react-virtuoso](https://virtuoso.dev) with the `Table` component.
```

Comment:
I think that it would be clearer with linking the GitHub repository. We tend to do this for all the projects so developers can land on "known ground".

```suggestion
In the following example, we demonstrate how to use [react-virtuoso](https://github.com/petyosi/react-virtuoso) with the `Table` component.
```